sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.aest.controller.getAllInProgressApplications.e2e-spec.ts

```diff
@@ -0,0 +1,181 @@
+    ]);
+
+    const endpoint = `/aest/application-offering-change-request/in-progress?page=0&pageLimit=10&sortField=status&sortOrder=${FieldSortOrder.DESC}&searchCriteria=Ministry Offering Change Test`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(
+        await getAESTToken(AESTGroups.BusinessAdministrators),
+        BEARER_AUTH_TYPE,
+      )
+      .expect(HttpStatus.OK)
+      .expect({
+        results: [
+          {
+            id: inProgressWithSABCApplicationOfferingChange.id,
+            applicationNumber:
+              applicationWithInProgressWithSABCApplicationOfferingChange.applicationNumber,
+            applicationId:
+              applicationWithInProgressWithSABCApplicationOfferingChange.id,
+            studyStartDate:
+              applicationWithInProgressWithSABCApplicationOfferingChange
+                .currentAssessment.offering.studyStartDate,
+            studyEndDate:
+              applicationWithInProgressWithSABCApplicationOfferingChange
+                .currentAssessment.offering.studyEndDate,
+            fullName: getUserFullName(
+              applicationWithInProgressWithSABCApplicationOfferingChange.student
+                .user,
+            ),
+            status:
+              inProgressWithSABCApplicationOfferingChange.applicationOfferingChangeRequestStatus,
+            createdAt: getISODateOnlyString(
+              applicationWithInProgressWithSABCApplicationOfferingChange.createdAt,
+            ),
+            studentId:
+              applicationWithInProgressWithSABCApplicationOfferingChange.student
+                .id,
+          },
+          {
+            id: inProgressWithStudentApplicationOfferingChange.id,
+            applicationNumber:
+              applicationWithInProgressWithStudentApplicationOfferingChange.applicationNumber,
+            applicationId:
+              applicationWithInProgressWithStudentApplicationOfferingChange.id,
+            studyStartDate:
+              applicationWithInProgressWithStudentApplicationOfferingChange
+                .currentAssessment.offering.studyStartDate,
+            studyEndDate:
+              applicationWithInProgressWithStudentApplicationOfferingChange
+                .currentAssessment.offering.studyEndDate,
+            fullName: getUserFullName(
+              applicationWithInProgressWithStudentApplicationOfferingChange
+                .student.user,
+            ),
+            status:
+              inProgressWithStudentApplicationOfferingChange.applicationOfferingChangeRequestStatus,
+            createdAt: getISODateOnlyString(
+              applicationWithInProgressWithStudentApplicationOfferingChange.createdAt,
+            ),
+            studentId:
+              applicationWithInProgressWithStudentApplicationOfferingChange
+                .student.id,
+          },
+        ],
+        count: 2,
```

Comment:
mm, Ministry should see all in progress requests irrespective of the institution right? so, `inProgressWithStudentApplicationOfferingChangeWithDifferentInstitution` details should also be returned right? correct me if I am wrong @andrepestana-aot @dheepak-aot @guru-aot @sh16011993

Reply:
Yes, you're right. I'm gonna fix that. Thanks!

Reply:
I fixed that. Thanks!